svc-aggregation/agmodel/system.go

```diff
@@ -538,6 +538,19 @@ func DeleteManagersData(key string) *errors.Error {
 	return nil
 }
 
+//DeleteLogServicesCollection will delete the LogServicesCollection entry from the database based on the  key
+
+func DeleteLogServicesCollection(key string) *errors.Error {
```

Comment:
Please write a generic method. or use the existing generic method if present

Reply:
addressed